delete.js

```diff
@@ -0,0 +1,12 @@
+const cloudinary = require("cloudinary").v2;
+
+cloudinary.config({
+  api_secret: "SWmuaA6jdhrtiRdmsoNQGCv4LzY",
+  api_key: 339361791885844,
+  cloud_name: "dn8ymrr2t",
```

Comment:
you need to delete your access key immediatly and create a new one. Then, save them in your .env.local as shown in my demo. Never publish these information in a public github repository!

Reply:
Yes was really stupid, I forgot that file.